src/pages/About.jsx

```diff
@@ -20,7 +20,9 @@ const developers = [
   {
     name: 'Gonzalo Lara',
     img: testPic,
-    description: 'Todo: about Gonzalo Lara',
+    description: 'My passion for computers has lead me to pursue a career in Software Engineering. I am currently '
+                 'a UWB CSS student completing my final year in the Computer Science and Software Engineering program. I am 24 years old, '
+                 'bilingual and currently a part of the DACA program. I intend to contribute as much as possible to building a better tomorrow.',
```

Comment:
Try using a [template literal](https://flaviocopes.com/javascript-template-literals/#multiline-strings)

Reply:
Ok I did that in the other pull request, this one can get deleted.

Reply:
I suggest leaving that change in this pull request. It adds extra noise to the other PR and merging will depend on when the other changes get approved.

Reply:
Got it, Well I included the changes to this pull if you want to take a look and merge or request further revisions.